### ISSUE TITLE
Always shown modals.

### ISSUE THREAD

**Issue by f** [bootstrap-vue/bootstrap-vue#416]
Hello,

I'm building a structure that uses routes as modal openers.

E.g. I make modal as child route:

```js
{
    path: '/users',
    component: Users,
    children: [
      { path: 'create', component: CreateUserModal },
    ],
  },
```

When I visit `/users/create` it opens the modal by mounting it. But when I do it, I need to make modals visible by default:

```js
mounted() {
  this.$refs.modal.show();
}
```

An attribute to make it visible by default would be nice for these kind of usage.

Maybe something like that may be nice:
```html
<b-modal always-open=true>...
```

**Comment by alexsasharegan** [comment]
So supposedly you can use the `v-model` directive to manipulate the visibility of the modal, but I can't get it to work.

**Comment by f** [comment]
Seems like there is something in code making this: https://github.com/bootstrap-vue/bootstrap-vue/blob/master/lib/components/modal.vue#L100 But I cannot make it work.

**Comment by pi0** [comment]
Yep, seems recent changes has broken v-model. Will investigate problem

**Comment by pi0** [comment]
Fixed in 0404c3428e251791afbe18c4446c062c6394c641 . v0.16.1.

**Comment by pi0** [comment]
See https://jsfiddle.net/pi0/orjf0g75/
